maths-club-pack/content/facilitator/russian-multiplication.md

```diff
@@ -0,0 +1,115 @@
+# Russian Multiplication
+
+## Introduction
+This is an interesting method to do multiplications – involving only halving, doubling and adding.  
+
+Work through the 22 x 19 example in a traditional way and then the method described. Both ways should give the answer 418.  
+
+Then try some other multiplications, e.g. 25 x 37. Students should be keen to test their own multiplications and see if this process always seems to work.
+
+
+## Solution  
+
+It will work for any two numbers. An insight into why it always works can be given by trying multiplications where one of the numbers is a power of two, e.g. 30 x 36 or 30 x 64 or 30 x 128. 
+
+
+## Extension
+
+**Understanding how Russian peasant multiplication is related to binary numbers**  
+
+(taken from http://mathforum.org/dr.math/faq/faq.peasant.html)   
+
+Binary numbers are numbers written in base two instead of base ten. This means that place value depends on powers of two instead of powers of ten: instead of ones, tens, and hundreds places, base two has a ones place, a twos place, a fours place, and so on. For example, fourteen in base two is 1110:  
```

Comment:
base two should include a hyperlink, i.e. [base two](https://mathforum.org/dr.math/faq/faq.bases.html)